src/app/pages/vm/vm-list/vm-list.component.ts

```diff
@@ -49,6 +50,11 @@ const noMemoryError = 'ENOMEM';
 export class VmListComponent implements EntityTableConfig<VirtualMachineRow> {
   title = this.translate.instant('Virtual Machines');
   queryCall = 'vm.query' as const;
+  queryCallOption: QueryParams<VirtualMachine, { extra: { retrieve_display_available_info: boolean } }> = [[], {
+    extra: {
+      retrieve_display_available_info: true,
```

Comment:
Hold off on this change. I've asked the middleware PR to remove the extra query params as I don't believe it's helpful. We should just show the `display_available` key by default.

Reply:
Okay, you don't need to specify these extra params in the query. The `display_available` key will be there by default now.